requirements.md

```diff
@@ -16,29 +16,46 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ## Web Context Support
 
 * It must possible to invoke SPC from an iframe. See [issue 68](https://github.com/w3c/secure-payment-confirmation/issues/68).
+
+Q: I understand that woudl also require PR API to be used from within the IFrame. Is that right?
+
 * It must possible to invoke SPC from within Payment Request API.
+
+Q: Shouldn't that be rather "It must be possible to invoke SPC via Payment Request API"? I understand the intention is to use the PR API as a way to invoke the SCP.
+
 * It must possible to invoke SPC from within a Payment Handler.
 
+Q: Does is mean the Payment Handler API woudl need to be extended by some new capabilities to trigger the SPC from within the PH code?
+
 ## Enrollment
 
 * It must be possible to enroll an SPC Credential outside of a transaction.
 * It must be possible to enroll an SPC Credential during a transaction. This enrollment should not prevent timely completion of the transaction.
+
+Q: Does that also imply that the enrolment during the transaction coudl somehow produce the SPC Attestation? Such capability woudl help to solve one of the difficult problems of getting conusmer authenticated for the first transaction.
```

Comment:
I don't think this is possible due to the trust model (not due to any technical reason!). The relying party has to already have authenticated the user in some form in order to enroll them, as otherwise how does it know they are who they say they are! (Stolen credit card, etc). So at the point where one can trust the user to be enrolled, they have likely already logged into their bank, or performed an OTP process, etc.

If I am wrong for some reason, we could in theory extend the enrollment process to also sign something.

Reply:
Good question. I am going to clarify this point to be: " It must be possible to enroll an SPC Credential following a prior (non-SPC) authentication during a transaction." 

I don't think the expectation was that SPC enrollment would take place without prior authentication. If that is a requirement, let's add it.

Reply:
Thanks. So the actual authentication would need to happen prior the new `PaymentRequest(...).show()`. Previously we discussed the concept of a fallback mechanism that would allow RP to leverage the 1PC context dialog to perform the authand then enroll the SPC credential following the successful authentication.